### DISCUSSION TITLE
Accidentally Pressing 'm' on xaringan Slides - Don't worry. You are not having a stroke. | /en/2020/04/xaringan-mirror/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1375] [Comments]
https://yihui.org/en/2020/04/xaringan-mirror/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
没办法，像 Netlify 这样免费又不受控制的资源就是容易被墙。多谢相告！

> ——原帖发布于 2020-09-16 01:35:16

**Comment by yyzeng** [comment]
学生可以用手机拍下答案页的照片，然后在手机上再 mirror 一次照片——这样他们就即看到答案又无需脖子受累啦。

**Reply by yihui** [reply]
是个人才！下课后先不要走。

**Reply by yyzeng** [reply]
刚试了下，发现手机照片没有mirror功能（至少在我的华为手机上没有），但……可以用自拍模式，得到的照片就无需mirror了。:smile:

**Reply by yihui** [reply]
要是有学生能把图片读进 R、将图片矩阵的列左右对调顺序、再把图片写出去看到答案的话，这原本的作业不做也罢，我给他满分。

手机拍照的同学我只能给他 82 分，剩下的 18 分以 666 的形式扣掉。
